data/tips.cfg

```diff
@@ -73,7 +73,7 @@
     source= _ "<i>― The Wesnoth Tactical Guide</i>"
 [/tip]
 [tip]
-    text= _ "Poisoned units will lose eight hitpoints every turn until they are one point from dying. They will then remain at death’s door until the poison is removed. Healers can prevent a poisoned unit from deteriorating, but the poison will not be removed unless the unit regenerates, or begins a turn in a village or next to a unit with the <i>Cures</i> ability."
+    text= _ "Poisoned units will lose eight hitpoints every turn until they are one point from dying. They will then remain at death’s door until the poison is removed. Healers can prevent a poisoned unit from deteriorating, but the poison will not be removed unless the unit regenerates, or begins a turn in a village, oasis, or next to a unit with the <i>Cures</i> ability."
```

Comment:
Should there be an "an" before "oasis"?

Reply:
Good point, thanks. I've changed the wording and pushed an amended commit